README.md

```diff
@@ -101,6 +101,21 @@ if err := f.Remove(); err != nil {
 }
 ```
 
+### Watch
+
+```go
+notifications := make(chan firego.Event)
+if err := f.Watch(notifications); err != nil {
+	log.Fatel(err)
+}
+defer func() { f.StopWatching() }()
+for {
+	event, ok := <-notifications
+	fmt.Println(event)
+	fmt.Println(ok)
```

Comment:
this loop would continue to spin forever, even if `notifications` is closed. let's add this at the end:

```go
if !ok {
	break
}
```